spec/game_spec.rb

```diff
@@ -0,0 +1,150 @@
+require_relative '../lib/game'
+
+describe Game do
+  describe '#add' do
+    subject(:game) { described_class.new }
+
+    it 'adds the coin to the bottom of the column' do
+      column = 3
+      game.add(column)
+      symbol = 'X'
+      expected = game.board.positions[5][column]
+      expect(expected).to eql symbol
+    end
+    it 'adds the second coin on top of last one' do
+      column = 3
+      game.add(column)
+      game.add(column)
+      symbol = 'X'
+      expected = game.board.positions[4][column]
+      expect(expected).to eql symbol
+    end
+    it "doesn't do anything if the column is full" do
```

Comment:
A good rule of thumb is "Don't test that something doesn't happen". If nothing is expected to happen in such a case, test is not needed.

I personally would throw an exception in this case.